src/bundle/Templating/Twig/SiteAccessNameExtension.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUiBundle\Templating\Twig;
+
+use Symfony\Component\Translation\TranslatorInterface;
+use Twig_Extension;
+use Twig_SimpleFunction;
+
+class SiteAccessNameExtension extends Twig_Extension
+{
+    /** @var \Symfony\Component\Translation\TranslatorInterface */
+    private $translator;
+
+    /**
+     * @param \Symfony\Component\Translation\TranslatorInterface $translator
+     */
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function getFunctions()
+    {
+        return [
+            new Twig_SimpleFunction(
+                'ezplatform_admin_ui_site_access_name',
```

Comment:
You don't have to prefix twig function with `ezplatform_admin_ui`, simple `siteaccess_label(identifier)` is enough. 

Also please note it's called `siteaccess` not `site access`. This also applies to the method names where you are using `siteAccess` term.